test/test_ES6.js

```diff
@@ -0,0 +1,88 @@
+import chai from 'chai';
+import { expect } from 'chai';
+import app from '../server';
+
+chai.use(require('chai-http'));
+
+
+// Simple test to check if mocha and travis were setup well
+function ret() {
+  return 1;
+}
+
+describe('Mocha and Travis are working', () => {
+  it('is a simple test that checks if moha and travis were setup properly', () => {
+    expect(ret()).to.equal(1);
+  });
+});
+
+// POST an enrty
+describe('Create an entry', () => {
+  it('Should create an entry', () => chai.request(app)
+    .post('/api/v1/entries')
+    .send({ title: 'Title', text: 'Text' })
+    .then((res) => {
+      expect(res).to.have.status(200);
```

Comment:
These test cases are not descriptive, refer to this https://medium.com/@yahelyechieli/5-tips-for-writing-great-javascript-unit-tests-86296ad2d997 for guideline

Reply:
Changed